.github/workflows/build-images.yml

```diff
@@ -28,6 +28,9 @@ jobs:
           - name: "mandrel builder images"
             modules: "quarkus-mandrel-builder-image"
             args: ""
+          - name: "mandrel builder images 21 and 22.0"
```

Comment:
nice, I had to do this because the agent didn't have enough disk space.

Reply:
I think it's OK to remove some of the images as well, e.g.:

* 21.3.2.0
* 22.0.0.2
* 22.1.0.0

are fairly dated.